project/src/Util/GetterServices.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
```

Comment:
You muse add a namespace for this class (and update the use in other classes) ;)